### DISCUSSION TITLE
SSL Proxy causes install to fail

### DISCUSSION THREAD

**Discussion by ironbract** [Security-Onion-Solutions/securityonion#6676] [Unsupported Versions] [answered]
On a fresh installation with Ubuntu 20.04, so-setup-network fails because the network is proxying access to github with an untrusted CA:

[WARNING] (01/10) Could not reach https://raw.githubusercontent.com/Security-Onion-Solutions/securityonion/master/KEYS, curl error code: 60

I know I need to add the proxy root CA to the system, and have installed the .pem for openssl (so it's now listed in /etc/ssl/certs).

The curl commands during the installation still fail to recognize the SSL proxy cert.

I'm obviously missing something so suggestions on what to do next are very welcome!

Thank you.

**Comment by tomjones1977** [comment, marked as answer]
How did you run the CA cert install?  There's a specific process needed to get the OS (and underlying apps) to use it.  

https://askubuntu.com/questions/73287/how-do-i-install-a-root-certificate

**Reply by ironbract** [reply]
Thank you! Those instructions are correct. I was actually performing basically the same thing, but I had installed the wrong CA certificate, so of course it still didn't match. PEBKAC.

Marking as answered.

**Comment by tomjones1977** [comment]
I did wonder if it might be that but I didn't dare ask - glad you sorted it
out though!
